aggregator_server/aggregator_server.cpp

```diff
@@ -191,7 +191,7 @@ class aggregator_server final : public zecale_proto::Aggregator::Service
     grpc::Status SubmitTransaction(
         grpc::ServerContext *,
         const zecale_proto::TransactionToAggregate *transaction,
-        proto::Empty *response) override
+        proto::Empty *) override
```

Comment:
Same question as above

Reply:
switch to `/*response*/` (and for context in this function)

Reply:
` proto::Empty * /*request*/) override` -> ` proto::Empty * /*response*/) override`